src/ad/abstract.js

```diff
@@ -0,0 +1,152 @@
+import { SortedListItem } from '../util/sorted-list-item'
+import { SortedList } from '../util/sorted-list'
+
+/**
+ * Abstract class for ads (i.e., {@link InLine} and {@link Wrapper}).
+ * @class AbstractAd
+ * @abstract
+ * @protected
+ * @extends SortedListItem
+ * @author Tim De Pauw <tim.depauw@zentrick.com>
+ * @copyright © 2018 Zentrick nv
```

Comment:
I noticed that `LICENSE` still has 2016 in it so please change that one as well.